scripts/main.js

```diff
@@ -0,0 +1,558 @@
+    { key: "H", code: 72 }, { key: "J", code: 74 }, { key: "K", code: 75 }, { key: "L", code: 76 }, { key: ":", code: 186 }, { key: "\"", code: 222 },
+    { key: "Enter", code: 13 }],
+    [{ key: "Shift", code: "ShiftLeft" }, { key: "Z", code: 90 }, { key: "X", code: 88 }, { key: "C", code: 67 }, { key: "V", code: 86 }, { key: "B", code: 66 },
+    { key: "N", code: 78 }, { key: "M", code: 77 }, { key: "<", code: 188 }, { key: ">", code: 190 }, { key: "?", code: 191 }, { key: "🡅", code: 38 },
+    { key: "Shift", code: "ShiftRight" }],
+    [{ key: "Ctrl", code: "ControlLeft" }, { key: "Alt", code: "AltLeft" }, { key: "Space", code: "Space" }, { key: "Alt", code: "AltRight" }, { key: "Ctrl", code: "ControlRight" },
+    { key: "🡄", code: 37 }, { key: "🡇", code: 40 }, { key: "🡆", code: 39 }, { key: "Win", code: 92 }]
+];
+
+const RU_KEYS_CAPS = [192, 81, 87, 69, 82, 84, 89, 85, 73, 79, 80, 65, 83, 68, 70, 71, 72, 74, 75, 76, 90, 88, 67, 86, 66, 78, 77, 219, 221, 186, 222, 188, 190, 191, 220];
+
+const EN_KEYS_CAPS = [81, 87, 69, 82, 84, 89, 85, 73, 79, 80, 65, 83, 68, 70, 71, 72, 74, 75, 76, 90, 88, 67, 86, 66, 78, 77];
+
+let keyPressed = true;
+let isEng = false;
+let isUpper = false;
+let infoBox = document.createElement("div");
+let infoText = document.createElement("span");
+let infoOS = document.createElement("span");
+let infoTextPlus = document.createElement("span");
+let infoShiftBtn = document.createElement("div");
+let infoShiftText = document.createElement("p");
+let infoCtrlBtn = document.createElement("div");
+let infoCtrlText = document.createElement("p");
+let inputText = document.createElement("div");
+let textArea = document.createElement("textarea");
+let buttonKey, key, lineBoard;
+let container = document.createElement("div");
+let keyBoard = document.createElement("div");
+let hotKey = false;
+let isCaps = false;
+
+createDomTree();
+
+function createDomTree() {
+    infoOS.innerText="Developed by Windows!";
+    infoText.innerText = "To change the language, press ";
+    infoTextPlus.innerText = "+";
+    infoShiftText.innerText = "Ctrl";
+    infoShiftBtn.classList.add("button-big");
+    infoCtrlText.innerText = "Shift";
+    infoCtrlBtn.classList.add("button-big");
+    container.classList.add("container");
+    keyBoard.classList.add("keyboard");
+    textArea.classList.add("inputbox");
+    infoBox.classList.add("infobox");
+    infoCtrlBtn.appendChild(infoCtrlText);
+    infoShiftBtn.appendChild(infoShiftText);
+    infoBox.appendChild(infoText);
+    infoBox.appendChild(infoShiftBtn);
+    infoBox.appendChild(infoTextPlus);
+    infoBox.appendChild(infoCtrlBtn);
+    infoBox.appendChild(infoOS);
+    inputText.appendChild(textArea);
+    container.appendChild(inputText);
+    container.appendChild(keyBoard);
+    container.appendChild(infoBox);
+    document.body.append(container);
+}
+
+document.body.onload=()=> {
+    if(localStorage.getItem("ENG")=="true"){
```

Comment:
try avoid abstract comparison. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators

Reply:
> try avoid abstract comparison.
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators

Thanks for your comment. Corrected and remembered.